research-hub-web/src/app/components/home/home.component.html

```diff
@@ -16,7 +16,7 @@
     <app-research-activity></app-research-activity>
   </div>
   <!-- Contact -->
-  <div class="grey-background">
+  <div id="contacts" class="grey-background contact-container">
```

Comment:
The contact-container class was removed with my css changes. It's empty css at the moment and I think it would be better if we left the class off the element. The id should be enough to select the element.

Reply:
updated